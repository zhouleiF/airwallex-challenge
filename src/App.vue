<template>
  <div id="app">
    <header>
      <p>BROCCOLI & CO.</p>
    </header>
    <section>
      <landing />
    </section>
    <footer>
      Made with heart in Melbourne.<br/>
      @ 2020 Broccoli & Co. All rights reserved.
    </footer>
  </div>
</template>
<script>
import Landing from '@/view/Landing'

export default {
  components: { Landing }
}
</script>

<style lang="scss" scoped>
#app {
  font-family: Avenir, Helvetica, Arial, sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  text-align: center;
  color: #2c3e50;
  font-size: 25px;
  display: flex;
  align-items: center;
  justify-content: center;
  height: 100vh;
}
header {
  position: fixed;
  width: 100%;
  top: 0;
  border-bottom: 1PX solid #000;
  p {
    width: 80%;
    text-align: left;
    margin: 0 auto;
    line-height: 90px;
  }
}
footer {
  display: flex;
  align-items: center;
  justify-content: center;
  width: 100%;
  position: fixed;
  bottom: 0;
  border-top: 1PX solid #000;
  line-height: 1.3;
  height: 110px;
}
</style>
