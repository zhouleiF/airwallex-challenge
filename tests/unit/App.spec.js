import { shallowMount } from '@vue/test-utils'
import App from '@/App.vue'

describe('App.vue', () => {
  it('renders App correctly', () => {
    const wrapper = shallowMount(App)
    expect(wrapper.element).toMatchSnapshot()
  })
})
