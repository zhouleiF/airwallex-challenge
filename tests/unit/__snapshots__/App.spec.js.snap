// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.vue renders App correctly 1`] = `
<div
  id="app"
>
  <header>
    <p>
      BROCCOLI & CO.
    </p>
  </header>
   
  <section>
    <landing-stub />
  </section>
   
  <footer>
    
    Made with heart in Melbourne.
    <br />
    
    @ 2020 Broccoli & Co. All rights reserved.
  
  </footer>
</div>
`;
